protocol/communities_messenger_test.go

```diff
@@ -574,6 +574,8 @@ func (s *MessengerCommunitiesSuite) TestImportCommunity() {
 	s.Require().NotNil(response)
 	s.Require().Len(response.Communities(), 1)
 	s.Require().Len(response.CommunitiesSettings(), 1)
+	s.Require().True(response.Communities()[0].Joined())
```

Comment:
@PascalPrecht isn't a bit dangerous accessing here with direct index without checking before that `if len(response.Communities()) > 0`?

Reply:
There's 

> 	s.Require().Len(response.Communities(), 1)

Two lines above that (L575), which should do the trick, no?

Reply:
@PascalPrecht yes, you're right, missed that somehow even I looked around this line.